webpack/config.js

```diff
@@ -154,48 +151,64 @@ module.exports = {
   },
 
   module: {
-    loaders: [
+    rules: [
       {
         test: /\.css$/i,
-        loader: "style-loader!css-loader!postcss-loader"
+        use: [
+          { loader: 'style-loader' },
+          { loader: 'css-loader' },
+          {
+            loader: 'postcss-loader',
+            options: {
+              ident: 'postcss',
+              plugins: function() {
+                return [
+                  require("postcss-import")(),
+                  require("postcss-cssnext")({ features: { rem: false } }),
+                  require('postcss-easings')(),
+                  require("postcss-browser-reporter")(),
+                  require("postcss-reporter")()
+                ];
+              }
+            }
+          }
+        ]
       },
       {
         test: /\.js$/i,
-        loader: 'babel',
-        exclude: /node_modules/
+        exclude: /node_modules/,
+        use: [
+          { loader: 'babel-loader' }
+        ]
       },
       {
         test: /\.mdx$/i,
-        loader: 'babel-loader!markdown-component-loader?passElementProps=true'
+        use: [
+          { loader: 'babel-loader' },
+          { loader: 'markdown-component-loader', options: { passElementProps: true } }
```

Comment:
this is so much nicer, I’m glad Webpack 2 is moving away from the query string nonsense.